### PR TITLE
acronym: update v1.5.0 to v1.6.0

### PR DESCRIPTION
**Pull request by lemoncurry**
---

Reviewer Resources:

[Track Policies](https://github.com/exercism/java/blob/master/POLICIES.md#event-checklist)

### CODE REVIEW

File: exercises/acronym/.meta/src/reference/java/Acronym.java [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
+            .toUpperCase();
     }
 
-    private String generateAcronym(String phrase) {
-        return Arrays.stream(phrase.split("[^a-zA-Z]"))
-                .filter(word -> !word.isEmpty()) // Remove empty strings from the result of phrase.split
-                .map(word -> word.substring(0, 1)) // Get the first character of each word
-                .collect(Collectors.joining()) // Concatenate the characters
-                .toUpperCase();
+    String getAcronym() {
```
Comment (FridaTveit) [inline exercises/acronym/.meta/src/reference/java/Acronym.java]: Why did you change the order of these methods? I would expect this one to come first since it's package private while `generateAcronym` is private 🙂

Comment (lemoncurry) [reply exercises/acronym/.meta/src/reference/java/Acronym.java]: Oh, I'm sorry, I think I mixed up your suggestion from a previous PR. I will change it!

Comment (FridaTveit) [reply exercises/acronym/.meta/src/reference/java/Acronym.java]: No problem, my comment might have been confusing! Thanks! 🙂
